packages/graph/hash_graph/lib/graph/src/store/postgres/ontology/read.rs

```diff
@@ -1,90 +1,131 @@
 use async_trait::async_trait;
 use error_stack::{IntoReport, Result, ResultExt};
 use futures::{StreamExt, TryStreamExt};
+use postgres_types::ToSql;
 use serde::Deserialize;
-use tokio_postgres::{GenericClient, RowStream};
+use tokio_postgres::GenericClient;
 
 use crate::{
-    ontology::types::uri::BaseUri,
+    ontology::{
+        types::{DataType, EntityType, LinkType, PropertyType},
+        AccountId, PersistedDataType, PersistedEntityType, PersistedLinkType,
+        PersistedOntologyIdentifier, PersistedPropertyType,
+    },
     store::{
         crud::Read,
-        postgres::{ontology::OntologyDatabaseType, parameter_list},
+        postgres::ontology::OntologyDatabaseType,
         query::{OntologyQuery, OntologyVersion},
         AsClient, PostgresStore, QueryError,
     },
 };
 
-async fn by_uri_by_version<T: OntologyDatabaseType>(
-    client: &(impl GenericClient + Sync),
-    uri: &BaseUri,
-    version: u32,
-) -> Result<RowStream, QueryError> {
-    client
-        .query_raw(
-            &format!(
-                r#"
-                SELECT schema
-                FROM {}
-                WHERE version_id = (
-                    SELECT version_id
-                    FROM ids
-                    WHERE base_uri = $1 AND version = $2
-                )
-                "#,
-                T::table()
-            ),
-            parameter_list([uri, &i64::from(version)]),
-        )
-        .await
-        .into_report()
-        .change_context(QueryError)
+fn parameter_list<const N: usize>(list: [&(dyn ToSql + Sync); N]) -> [&(dyn ToSql + Sync); N] {
```

Comment:
Seems added back to ontology, is it still around in the parent module? Or is it not required in knowledge anymore?

Reply:
This was a rebase issue, thanks for flagging: 01b433f0c6688d51973ff59a0afd4117423a91f0